docs/INSTALL.rst

```diff
@@ -146,6 +146,16 @@ To enable the database backend, make sure the **[extensions]** section of
 
   hil.ext.auth.database =
 
+Keystone Backend
+^^^^^^^^^^^^^^^^
+
+To enable the Keystone backend, make sure the **[extensions]** section of
+``hil.cfg`` contains::
+
+  hil.ext.auth.keystone =
+
+Visit the `Keystone configuration guide <http://hil.readthedocs.io/en/latest/keystone-auth.html>`_ for more information.
```

Comment:
Most of the rest of the docs just reference the file in `docs/`, seems a bit odd to me to link to the web for just this one.